src/web/core/src/main/java/org/geoserver/web/ServiceLinkDescription.java

```diff
@@ -35,18 +35,18 @@ public class ServiceLinkDescription implements Serializable, Comparable<ServiceL
     private final String layer;
 
     public ServiceLinkDescription(
-            String service, Version version, String link, String workspace, String layer) {
-        this(service, version, link, workspace, layer, null);
+            String serviceType, Version version, String link, String workspace, String layer) {
+        this(serviceType, version, link, workspace, layer, null);
     }
 
     public ServiceLinkDescription(
-            String service,
+            String serviceType,
             Version version,
             String link,
             String workspace,
             String layer,
             String protocol) {
-        this.service = service.toLowerCase();
+        this.service = serviceType.toLowerCase();
```

Comment:
I think (??) this might be a bit inconsistent with rest of the code (which seems to be based on uppercase).  I don't think this will be an issue since you use `.equalsIgnoreCase`...

Reply:
Thanks for your feedback David; can you think of a good location we could add string constants for these values?